### PR TITLE
fix: Fix errors introduced in previous 2 PRs

### PR DESCRIPTION
**Pull request by BYK**
The target creation snippet had an extra `\`, and the duplicate issue checker was referencing a non-existent `$subdirectory` variable due to a c/p error.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -60,16 +60,25 @@ runs:
...
         GITHUB_API_TOKEN: ${{ github.token }}
+    - name: Set issue title for publish request
+      shell: bash
+      run: |
+        if [[ '${{ inputs.path }}' == '.' ]]; then
+          subdirectory='';
+        else
+          subdirectory='/${{ inputs.path }}';
+        fi
+        echo "ISSUE_TITLE=publish: $GITHUB_REPOSITORY$subdirectory@$RELEASE_VERSION" >> $GITHUB_ENV
```
Comment (chadwhitacre) [inline action.yml:76]: Does GITHUB_ENV want quotes around env values here?

Comment (BYK) [reply action.yml:76]: This doesn't say anything about it: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

File: action.yml [hunk 2]
```diff
@@ -60,16 +60,25 @@ runs:
...
     - name: Check existing publish requests
       id: existing-request
       shell: bash
       run: |
-        issue_count=$(gh issue list -S "'publish: $GITHUB_REPOSITORY$subdirectory@$RELEASE_VERSION' in:title" | wc -l)
+        issue_count=$(gh issue list -S "'$ISSUE_TITLE' in:title" | wc -l)
```
Comment (chadwhitacre) [inline action.yml:81]: What happens if ISSUE_TITLE contains a `'`?

File: action.yml [hunk 3]
```diff
@@ -60,16 +60,25 @@ runs:
         cd '${{ inputs.path }}'
         npm install -g @sentry/craft@${{ inputs.craft_version }}
         craft prepare --no-input "${{ env.RELEASE_VERSION }}"
-        targets=$(craft targets | jq -r '.[]|"  [ ] \\(.)"')
```
Comment (chadwhitacre) [inline action.yml:63]: 👍

File: action.yml [hunk 4]
```diff
@@ -60,16 +60,25 @@ runs:
...
+        targets=$(craft targets | jq -r '.[]|"  [ ] \(.)"')
```
Comment (chadwhitacre) [inline action.yml:63]: 👍

File: action.yml (hunk 2 above)
Comment (BYK) [reply action.yml:81]: I mean we craft the title and no pieces of it should contain a `'`, right?

File: action.yml (hunk 1 above)
Comment (BYK) [reply action.yml]: Okay, quoted based on what we saw on https://github.com/c-py/action-dotenv-to-setenv

File: action.yml [hunk 5]
```diff
@@ -53,42 +53,52 @@ runs:
...
         echo "::set-output name=last::$(gh api repos/:owner/:repo/releases/latest | jq -r .tag_name)"
     - name: Request publish
-      if: steps.existing-request.outputs.count == '0'
       shell: bash
       run: |
+        # TODO(byk): Remove these lines when gh is updated in GHA
+        curl -L 'https://github.com/cli/cli/releases/download/v1.8.0/gh_1.8.0_linux_amd64.deb' > gh.deb
+        sudo dpkg -i gh.deb
+        rm gh.deb
+
```
Comment (chadwhitacre) [inline action.yml:88]: You and the bleeding edge. :P

**Comment by BYK** [comment]
Okay this is now ready: https://github.com/getsentry/release-tester/runs/2254078817?check_suite_focus=true

**Review by chadwhitacre — APPROVED** [review]

**Pull request merged**